code/index.html

```diff
@@ -52,11 +100,17 @@ <h1>Does the person have</h1>
       <div class="win-or-lose">
         <img
           class="guess-who-icon"
-          src="images/guess-who-bubble.png"
+          src="imagesnew/guessWhoLogoNew.png"
           alt="Guess Who"
         />
         <h1 id="winOrLoseText"></h1>
         <button type="button" id="playAgain" class="filled-button">PLAY AGAIN</button>
+        <section class="count-section">
+          <h2>Your number of guesses was:
+            <span class="guess-container" id="win-or-lose-guess-container">0</span>
+          </h2>
+        </section>
+        <h2 id ="winOrLoseTimer"></h2>
```

Comment:
It's a great addition to show the user how many guesses they ended up making once the game has finished, great job! I'd say here you could have tried adding the variable name for the total amount of guesses straight into the Javascript with a template literal? Just a little tip ☺️